pkg/didmethod/httpuniversal/resolver.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package httpuniversal
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	errors "golang.org/x/xerrors"
+)
+
+//HTTPClient abstracts away http client library
+type HTTPClient interface {
+	Get(url string) (body []byte, err error)
+}
+
+//StandardHTTPClient is based on http.Client
+type StandardHTTPClient struct {
+	client *http.Client
+}
+
+//Get implements HTTP Get operation on standard http client
+func (shc StandardHTTPClient) Get(url string) (body []byte, err error) {
```

Comment:
Although not really needed in this case, let's always use pointer receivers for consistency (see [Go FAQ](https://golang.org/doc/faq#methods_on_values_or_pointers)).

Reply:
Agree, fixed.